src/service/vsync-impl.js

```diff
@@ -441,14 +441,19 @@ function callTaskNoInline(callback, state) {
   return true;
 }
 
-
 /**
  * @param {!Window} window
  * @return {!Vsync}
  */
+export function getVsyncService(window) {
+  installVsyncService(window);
+  return getService(window, 'vsync');
+}
```

Comment:
Ok, how about replacing this method with VsyncServiceForTesting (since we need it in one place) and changing the rest of the callers to use vsyncFor from vsync.js?